third_party/gpus/crosstool/clang/bin/crosstool_wrapper_driver_rocm.tpl

```diff
@@ -145,7 +145,7 @@ def InvokeHipcc(argv, log=False):
   undefines = GetOptionValue(argv, 'U')
   undefines = ''.join([' -U' + define for define in undefines])
   std_options = GetOptionValue(argv, 'std')
-  hipcc_allowed_std_options = ["c++11"]
+  hipcc_allowed_std_options = ["c++14"]
```

Comment:
Can you elaborate on why c++14 is required?
@whchung for awareness

Reply:
it's due to rocblas uses c++14 feature in their public accessible headers. even though rocblas needs justification why c++14 feature must be used, it would be short-term fix to let tensorflow buildable.

Reply:
RocBLAS has already fixed this issue, please try the tip of its develop branch.

Reply:
RocBLAS fixed it in master branch by https://github.com/ROCmSoftwarePlatform/rocBLAS/commit/af2b90d643793a9f357f732a9b26f64534746ec2 but not in develop branch.

I asked rocblas to cherrypick the fix to develop branch.

I have to abandon this PR since I found switching to c++14 causes other issues due to rocprim is not compatible with c++14 
https://github.com/ROCmSoftwarePlatform/rocPRIM/blob/c5c8b35d062cf18acfa82d7e801fad5222a05cb2/rocprim/include/rocprim/types/tuple.hpp#L386 

::rocprim::index_sequence<Indices...> not defined with c++14